packages/kbn-test/src/functional_tests/tasks.js

```diff
@@ -25,9 +25,10 @@ in another terminal session by running this command from this directory:
  * @param {string[]} configPaths         Array of paths to configs
  * @param {object}   options             Optional
  * @param {Log}      options.log         Optional logger
```

Comment:
__nit:__ could you please make `options.log` a `@property` as well? And since we don't align argument names anymore, we can get rid of unnecessary spaces between argument type and its name.

Reply:
thanks, done.